src/components/ThemeContainer/__snapshots__/ThemeContainer.test.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<ThemeContainer /> renders connected component correctly 1`] = `null`;
```

Comment:
If this were a HOC, then these snapshots could go away, as they don't really serve a purpose right now.